app/src/main/java/com/ruslanchikbananchik/rmma/interface/RetrofitServices.kt

```diff
@@ -0,0 +1,11 @@
+package com.ruslanchikbananchik.rmma.`interface`
+
+import com.ruslanchikbananchik.rmma.model.Fact
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface RetrofitServices {
+    @GET("?count=3")
+
```

Comment:
Codestyle violation 

https://kotlinlang.org/docs/coding-conventions.html#annotations
remove this empty line